substrate/frame/revive/src/storage.rs

```diff
@@ -70,12 +65,6 @@ pub struct ContractInfo<T: Config> {
 	/// We need to store this information separately so it is not used when calculating any refunds
 	/// since the base deposit can only ever be refunded on contract termination.
 	storage_base_deposit: BalanceOf<T>,
-	/// Map of code hashes and deposit balances.
-	///
-	/// Tracks the code hash and deposit held for locking delegate dependencies. Dependencies added
-	/// to the map can not be removed from the chain state and can be safely used for delegate
-	/// calls.
-	delegate_dependencies: DelegateDependencyMap<T>,
 	/// The size of the immutable data of this contract.
 	immutable_data_len: u32,
```

Comment:
We don't care about breaking existing contracts on Westend as of right now. We will do a reset of the pallet once we are stable.

What we can't except is panicing the runtime, though. Removing data will not lead to a panic. The trailing data will be ignored and `immutable_data_len` will have a wrong value (most often `0`).

However, I need to verify this. Couldn't find out if the `StorageMap` panics if trailing data is present.

Reply:
I assume the failing of the Westend Assethub Migration CI job contradicts my assumoption. Removing this field breaks decoding of the storage item. Need to think of another solution unless we want to provide a storage migration.

Reply:
Okay I added a migration to deal with this.